test/utils/pickers-utils.tsx

```diff
@@ -70,18 +72,25 @@ export function wrapPickerMount(mount: (node: React.ReactNode) => import('enzyme
 
 export function createPickerRenderer({
   locale,
+  adapterName,
```

Comment:
We can run the entire test codebase with any of the adapters. But only some of them are relevant to be run on various adapters. That's why I add this parameter. By priority, tests run with the adapter provided by:

- adapterName
- test:unit paramter
- date-fns (default one)